package.json

```diff
@@ -16,6 +17,7 @@
     "@release-it-plugins/lerna-changelog": "^5.0.0",
     "@release-it-plugins/workspaces": "^3.2.0",
     "release-it": "^15.5.0",
+    "ember-source": "^6.0.0",
```

Comment:
@NullVoxPopuli should we just install it in addon workspace?

Reply:
we don't want to because that can (and usually does) mess up how peers resolve